<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoControlWindow</class>
 <widget class="QWidget" name="AutoControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(114, 159, 207);</string>
  </property>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stop_button
{
border-image: url(:/images/stop_unpressed.png);
}

#stop_button:pressed
{
border-image: url(:/images/stop_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#start_button
{
border-image: url(:/images/start_unpressed.png);
}

#start_button:pressed
{
border-image: url(:/images/start_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="add_button">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#add_button
{
border-image: url(:/images/add_unpressed.png);
}
#add_button:pressed
{
border-image: url(:/images/add_pressed.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Add Command</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#delete_button
{
border-image: url(:/images/delete_unpressed.png);
}
#delete_button:pressed
{
border-image: url(:/images/delete_pressed.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Delete Command</string>
     </property>
    </widget>
    <widget class="QListWidget" name="command_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);
background-color: rgb(46, 52, 54);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Repeat</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>49</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="position_y">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_pos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Lock Position X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>Lock Position Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="position_x">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration Page</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(114, 159, 207);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Select Mode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>  Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
